### ISSUE TITLE
Generated struct limit method incorrect...

### ISSUE THREAD

**Issue by jfallows** [reaktivity/nukleus-maven-plugin#45]
... when last field is fixed length and after a variable length field.

```
struct FixedAfterVariable
{
    string16 variable;
    int32 fixed;
}
```

Generated `limit()` method incorrectly returns `variableRO.limit()`.

**Comment by cmebarrow** [comment]
@jfallows Please close if the issue is resolved

**Comment by jfallows** [comment]
Resolved by #47.
